test/assets/preset-mini-targets.ts

```diff
@@ -962,6 +962,9 @@ export const presetMiniTargets: string[] = [
 
   // variants - multiple parents
   'sm:lt-lg:p-10',
+
+  // shortcuts
+  'btn',
```

Comment:
Can we move to another test suite so it could be clear it's testing the shortcuts instead of the preset?

I'd also want to test `hover:btn` and `focus:btn`